packages/babel-types/src/index.js

```diff
@@ -400,6 +400,49 @@ export function buildMatchMemberExpression(
 }
 
 /**
+ * Add comment of certain type to a node.
+ */
+
+export function addComment(
+  node: Object,
+  type: string,
+  content: string,
+  line?: boolean,
+): Object {
+  addComments(node, type, [
+    {
+      type: line ? "CommentLine" : "CommentBlock",
+      value: content,
+    },
+  ]);
+}
+
+/**
+ * Add comments of certain type to a node.
+ */
+
+export function addComments(
+  node: Object,
+  type: string,
+  comments: Array,
+): Object {
+  if (!comments || !node) return;
+
+  const key = `${type}Comments`;
+
+  if (node[key]) {
+    if (type === "leading") {
+      node[key] = comments.concat(node[key]);
+    } else {
+      node[key] = node[key].concat(comments);
+    }
+  } else {
+    node[key] = comments;
+  }
+  return node;
+}
+
+/**
```

Comment:
That's just a move from `babel-traverse` to `babel-types` right? Why?

Reply:
yeah, that was just moved - `babel-traverse` methods are proxying to those in `babel-types` in my PR

generated babel helpers are just nodes and not a nodePaths so I couldnt use `NodePath.prototype.addComment`, but as it was just unpacking the underlaying node I've moved its implementation to the `babel-types` package

so now we can add comments to nodes easily and ive used it to annotate generated babel helpers